app/components/Videos/VideoDetails.tsx

```
@@ -77,7 +77,7 @@ export default function VideoDetails({ asset }: VideoDetailsProps) {
     const conProps = {
       ...(asset.playbackPolicy && {
         accessKey: generateAccessKey(
-          activeAccount?.address,
+          activeAccount!.address,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)